saleor/settings.py

```diff
@@ -407,17 +426,6 @@ def get_bool_from_env(name, default_value):
 
 COUNTRIES_OVERRIDE = {"EU": "European Union"}
 
-
-def get_host():
-    from django.contrib.sites.models import Site
-
-    return Site.objects.get_current().domain
-
-
-PAYMENT_HOST = get_host
-
-PAYMENT_MODEL = "order.Payment"
-
```

Comment:
Correct me if I'm wrong, but I couldn't find any usage of the settings `PAYMENT_HOST` and `PAYMENT_MODEL` in the code. So, I've removed them because they seemed unnecessary.